### ISSUE TITLE
Searching Chandra data with astroquery.esasky.query_object_maps returns wrong objects

### ISSUE THREAD

**Issue by clgreenwell** [astropy/astroquery#2003]
When searching for Chandra measurements with astroquery.esasky.query_object_maps, the results returned can be several degrees off the input coordinates. I tested the same program with XMM instead and it works fine. Here is a working example:

```python
from astropy.coordinates import SkyCoord
from astroquery.esasky import ESASky
from astropy import units as u
from astropy.table import Table

tel = 'chandra'

input_ra = 1.77185950000001
input_dec = 20.207457
input_coords = SkyCoord(ra=input_ra*u.deg, dec=input_dec*u.deg, frame='icrs')
results = ESASky.query_object_maps(input_coords, tel)

print('input coordinates')
print(input_ra,input_dec)
print()

# create a table of the new results
new_results = Table(results[0])

print('output coordinates:')
print()

for j in range(len(new_results)):
    print(new_results['s_ra'][j],new_results['s_dec'][j])
    output_coords = SkyCoord(ra=new_results['s_ra'][j]*u.deg, dec=new_results['s_dec'][j]*u.deg, frame='icrs')
    sep = input_coords.separation(output_coords)
    print(f'Separation: {sep}')
    print()
```

Which outputs:
>input coordinates
1.77185950000001 20.207457
output coordinates:
350.5374527924661 40.88206224433065
Separation: 22.77901873075805 deg
356.0059298940373 26.04901132346725
Separation: 7.8866344575171015 deg
10.35709206391789 25.50027470632677
Separation: 9.514003631667578 deg
358.9436220532873 31.8862458793601
Separation: 11.950398759469767 deg

**Comment by keflavich** [comment]
I can reproduce this.  That's rather weird behavior.

A brief tip: `results[0]` is already a table, so you don't need to make it into one.  Also, you can loop over the table rows instead of their indices, e.g.:
```python
for row in results[0]:
    print(row['s_ra'], row['s_dec'])
```

**Comment by clgreenwell** [comment]
Thanks - yes, it is strange. Glad it's not just me.

Good point about the table. The loop over indices is partly because it's a simplified example of a longer script, but it might be that I can improve the main code with that. Thanks for the tips!

**Comment by imbasimba** [comment]
Thanks @clgreenwell for opening this issue! It totally slipped past me, sorry about that. Anyway, I cannot reproduce the issue. Most likely the issue originated in the ESASky backend or possibly the Chandra archive, and it has been fixed at some time in the last 1.5 years. 

The particular coordinates in your test shouldn't have any data. I also tested in an area where there is Chandra data, and it now seems like the ESASky module is returning the correct rows.
